### PR TITLE
WEBAPP-191: Add link to categories home

### PR DESCRIPTION
**Pull request by maxammann**

### CODE REVIEW

File: src/modules/layout/containers/GeneralHeader.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import React from 'react'
```
Comment (klinzo) [inline src/modules/layout/containers/GeneralHeader.js]: why did you move GeneralHeader to the containers package? It does not connect to the store nor does it use withFetcher or translations

Comment (klinzo) [reply src/modules/layout/containers/GeneralHeader.js]: Okay, but then LocationHeader and GeneralHeader should also be containers because Header is connecting to the state

Comment (Schedulaar) [reply src/modules/layout/containers/GeneralHeader.js]: Hmm.. yap... Do you think, we need a better definition of when to call a ReactComponent a container and when to call it a component? Or does this distinction still make sense in our project at all? 🤔
At least I like, that we have some distinction between complex ReactComponents, that can modify the state, vs dumb / just visual components...

**Comment by maxammann** [comment]
@Schedulaar @klinzo my solution is now to make Header a component. So I added Headroom to LocationLayout and RouterFragment which are already containers

File: src/modules/layout/components/Header.js [hunk 2]
```diff
@@ -2,27 +2,23 @@ import React from 'react'
...
  * of the Header.
  * Uses Headroom to save space when scrolling.
  */
-export class Header extends React.Component {
+class Header extends React.Component {
   static propTypes = {
     navigationItems: PropTypes.arrayOf(PropTypes.instanceOf(HeaderNavigationItem)).isRequired,
     actionItems: PropTypes.arrayOf(PropTypes.instanceOf(HeaderActionItem)).isRequired,
-    smallViewport: PropTypes.bool.isRequired
+    logoHref: PropTypes.string.isRequired
```
Comment (Schedulaar) [inline src/modules/layout/components/Header.js]: Add viewport here
